### PR TITLE
Update Profile docs (part II)

### PR DESCRIPTION
**Pull request by yangyansong-adbe**
- https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/691#discussion_r804001432
- https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/691#discussion_r804007594
Address above review comments in a new PR.

### CODE REVIEW

**Review by calebk1 — COMMENTED** [review]
Looks good!

Just, change it from "Examples" to "Example", please!

File: foundation-extensions/profile/profile-api-references.md [hunk 1]
```diff
@@ -338,6 +379,117 @@ private void handleCallback(NSDictionary content, NSError error)
...
+```
+
+**Example**
+
+```swift
+import AEPUserProfile
+
+...
+MobileCore.registerExtensions([UserProfile.self])
+```
```
Comment (emdobrin) [inline foundation-extensions/profile/profile-api-references.md:428]: For consistency with the ACP tab, can we also include the didFinishLaunchingWithOptions in these AEP samples?

File: foundation-extensions/profile/profile-api-references.md [hunk 2]
```diff
@@ -501,6 +663,14 @@ UserProfile.removeUserAttributes(Arrays.asList("username", "usertype"));
 
 ### Objective-C
 
+**Syntax**
+
+```objectivec
++ (void) removeUserAttributesWithAttributeNames:(NSArray<NSString *> * _Nonnull)
```
Comment (emdobrin) [inline foundation-extensions/profile/profile-api-references.md:669]: this syntax does not seem to match the example below, which one is correct?

File: foundation-extensions/profile/profile-api-references.md [hunk 3]
```diff
@@ -811,30 +995,36 @@ UserProfile.updateUserAttributes(attributeDict: profileMap)
 
 ### Objective-C
 
+**Syntax**
+
+```objectivec
++ (void)updateUserAttributesWithAttributeDict:(NSDictionary<NSString *,id> * _Nonnull)
```
Comment (emdobrin) [inline foundation-extensions/profile/profile-api-references.md:1001]: Same comment here, the example does not match this syntax

File: foundation-extensions/profile/profile-api-references.md (hunk 1 above)
Comment (yangyansong-adbe) [reply foundation-extensions/profile/profile-api-references.md]: Sure, I will update the AEP tab.

File: foundation-extensions/profile/profile-api-references.md (hunk 2 above)
Comment (yangyansong-adbe) [reply foundation-extensions/profile/profile-api-references.md:679]: The API name should be `removeUserAttributesWithAttributeNames`, I will update the example code. 
https://github.com/adobe/aepsdk-compatibility-ios/blob/12c5e8681905140f0a09e20e2339fed85f4d9ea8/ACPUserProfile/ACPUserProfile.m#L36

File: foundation-extensions/profile/profile-api-references.md (hunk 3 above)
Comment (yangyansong-adbe) [reply foundation-extensions/profile/profile-api-references.md:1011]: The example code is fixed.

File: foundation-extensions/profile/profile-api-references.md (hunk 2 above)
Comment (emdobrin) [reply foundation-extensions/profile/profile-api-references.md:679]: :(

**Review by emdobrin — APPROVED** [review]

**Pull request merged**